src/tribler-gui/tribler_gui/widgets/videoplayerpage.py

```diff
@@ -57,23 +64,26 @@ def initialize_player(self):
         self.mediaplayer = self.instance.media_player_new()
         self.window().video_player_widget.should_hide_video_widgets.connect(self.hide_video_widgets)
         self.window().video_player_widget.should_show_video_widgets.connect(self.show_video_widgets)
-        self.window().video_player_position_slider.should_change_video_position.connect(
-            self.on_should_change_video_time
-        )
+        #self.window().video_player_position_slider.should_change_video_position.connect(
+        #    self.on_should_change_video_time
+        #)
         self.window().video_player_volume_slider.valueChanged.connect(self.on_volume_change)
         self.window().video_player_volume_slider.setValue(self.mediaplayer.audio_get_volume())
         self.window().video_player_volume_slider.setFixedWidth(0)
 
         self.window().video_player_play_pause_button.clicked.connect(self.on_play_pause_button_click)
+        self.window().video_player_stop_button.clicked.connect(self.on_stop_button_click)
```

Comment:
Nice! We really needed a stop button. At least now we get out of VOD mode.

Reply:
cool